src/plugins/intel_cpu/src/nodes/concat.cpp

```diff
@@ -40,6 +40,9 @@ bool Concat::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std
             errorMessage = "Node is not an instance of the Concat operation.";
             return false;
         }
+        if (concatOp->get_output_element_type(0) == ov::element::string) {
+            return false;
+        }
```

Comment:
```suggestion
        } else if (concatOp->get_output_element_type(0) == ov::element::string) {
            errorMessage = "Giver error message for not supported type"; // ??
            return false;
        }
```

Reply:
This part is responsible for fallback to the reference, it doesn't throw any error.